js/thumbnail.js

```diff
@@ -0,0 +1,28 @@
+const thumbnailTemplate = document
+	.querySelector('#picture').content
+	.querySelector('.picture');
+
+const container = document.querySelector('.pictures');
+
+const createThumbnail = ({ comments, description, likes, url }) => {
+	const thumbnail = thumbnailTemplate.cloneNode(true);
+
+	thumbnail.querySelector('.picture__img').src = url;
+	thumbnail.querySelector('.picture__img').alt = description;
```

Comment:
Я всё-таки советую DRY
```suggestion
	const image = thumbnail.querySelector('.picture__img');
	image.src = url;
	image.alt = description;
```